Modix.Data/Models/Core/DesignatedChannelType.cs

```diff
@@ -24,6 +24,10 @@ public enum DesignatedChannelType
         /// <summary>
         /// Defines a channel that is not subject to auto-moderation behaviors of the moderation feature.
         /// </summary>
-        Unmoderated
+        Unmoderated,
+        /// <summary>
+        /// Defines a channel to which starred messages are sent.
+        /// </summary>
+        Starboard
```

Comment:
There's a corresponding enum for the web interface as well that needs to be edited.
https://github.com/discord-csharp/MODiX/blob/master/Modix/ClientApp/src/models/moderation/ChannelDesignation.ts